library/psa_crypto_driver_wrappers.c

```diff
@@ -0,0 +1,378 @@
+/*
+ *  Functions to delegate cryptographic operations to an available
+ *  and appropriate accelerator.
+ *  Warning: auto-generated file.
+ */
+/*  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include "psa_crypto_core.h"
+#include "psa_crypto_driver_wrappers.h"
+#include "mbedtls/platform.h"
+
+#if defined(MBEDTLS_PSA_CRYPTO_DRIVERS)
+
+/* Include test driver definition when running tests */
+#if defined(PSA_CRYPTO_DRIVER_TEST)
+#undef PSA_CRYPTO_DRIVER_PRESENT
```

Comment:
I was imagining each driver declaration contributing this block, and was trying to work around redefinition errors. Will change to a #ifndef instead of explicit redefinition.

Reply:
```
#define FOO whatever
#define FOO whatever
```
is valid C as long as the two `whatever` are identical token sequences. A picky compiler could warn about this but as far as I recall we do have precedents of defining the same macro twice and nobody complained. (Somebody did complain once, but it turned out that the definitions didn't match in that particular configuration, which made the compiler's complaint perfectly justified.)

Reply:
Sure, but you pointed out someone could declare it with a value, whereas I'm only using the 'defined' property. Meaning I'll stick with the ifndef to make sure whatever value it gets declared with is retained.
Feel free to change when you get around to autogeneration.